frontend/src/metabase/visualizations/visualizations/CartesianChart/padding.ts

```diff
@@ -0,0 +1,21 @@
+import type { MantineTheme } from "metabase/ui";
+
+export const getChartPadding = ({
+  theme,
+  isQueryBuilder,
+}: {
+  isQueryBuilder?: boolean;
+  theme: MantineTheme;
+}) => {
+  const { padding } = theme.other.cartesian;
+
+  if (padding) {
+    return padding;
+  }
+
+  if (isQueryBuilder) {
+    return "1rem 1rem 1rem 2rem";
```

Comment:
Can you add a comment linking to https://github.com/metabase/metabase/pull/17552#issuecomment-904945088?

This has been moved so many times and it took me awhile to figure out why we need this.

By the way, I just checked and this still holds true.